spec/sample-data/valid/gazette-notice-01.json

```diff
@@ -0,0 +1,46 @@
+{
+  "issue": {
+    "publication": {
+      "publisher": {
+        "name": "Commissioners of Public Works in Ireland",
+        "url": "http://www.opw.ie/"
+      },
+      "jurisdiction_code": "ie",
+      "title": "Iris Oifigiuil",
+      "url": "http://www.irisoifigiuil.ie/"
+    },
+    "id": "1",
```

Comment:
Wonder if we should avoid #id attribute, as if we extend the attributes to the ruby objects this could have undesired consequences